tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
-coverage
+coverage>=4.4.0
 ddt
 mock
-pytest
+pytest>=3.6.0
```

Comment:
This does not pin it to max, but requires a min..?!

However, pytest-cov should be fixed.